src/dbspro/cli/run.py

```diff
@@ -37,28 +37,24 @@ def main(args):
 
         # Check if file has the correct extension
         if not str(args.fastq).endswith(accepted_file_ext):
-            raise FileNotFoundError(f'File {args.fastq} is not accepted input.')
+            raise FileNotFoundError(f"File {args.fastq} is not accepted input.")
 
         # Create symbolic link to file in output directory.
         os.symlink(args.fastq, f"{args.directory}/{accepted_file}")
-        logging.info('Creating symbolic link for input file in output directory.')
+        logging.info("Creating symbolic link for input file in output directory.")
 
     # Create dict containing the paramaters to be passed to the snakefile.
     configs_dict = {
-        'dbs_cluster_dist': args.dbs_cluster_dist,
-        'abc_cluster_dist': args.abc_cluster_dist,
-        'filter_reads': args.filter_reads,
-        'abc_sequences': args.abcs,
-        'handles': args.handles,
-        'dbs_len': args.dbs_len,
-        'umi_len': args.umi_len,
+        "dbs_cluster_dist": args.dbs_cluster_dist,
+        "abc_cluster_dist": args.abc_cluster_dist,
+        "filter_reads": args.filter_reads
     }
```

Comment:
Note that these setting will overwrite the settings in the `dbspro.yaml` file. Including with the default values. I say we remove these options entirely and just use the config file.

Reply:
Which is why I removed them and only kept the options not specified in dbspro.yaml :P

Reply:
That is true, I missed that the `dbs_cluster_dist`, etc. options are only in the `run` options not in the config file. However I think it would be prudent to move these to the config file to as having different settings on two places is quite confusing in my mind :)

Reply:
Sure, that makes sense!